windowsdoc.md

```diff
@@ -0,0 +1,67 @@
+## **Windows Compatible Cwltool**  
+Windows Compatible cwltool means that now you can create and run your workflows usinng cwltool on windows under a docker container.
+On windows, all tools and workflows will be executed inside a linux based docker container. If a Docker Container is not provided explicitly(using [Docker Requirement](http://www.commonwl.org/v1.0/CommandLineTool.html#DockerRequirement)) by user, we use a minimal, posix compliant Alpine Docker image which has Bash pre installed as fallback(Default) [Docker Container](https://github.com/frol/docker-alpine-bash).  
+You can also provide your own Default Docker Container using `--default-container` argument.
+
+### ***Requirements***:  
+* python 2 or 3, Yes Now we support python 3 too.  
```

Comment:
Capital on Python here and I would simplify this line to just "Python 2 or 3", otherwise this document will get outdated